guides/common/modules/con_content-flow-in-projectname.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Content-Flow-in-{ProjectNameID}_{context}"]
+= Content Flow in {ProjectName}
+
+Content in {ProjectName} flows from _external content sources_ to _{ProjectServer}_.
+The {ProjectServer} is the central component where you manage the content lifecycle.
+_{SmartProxyServers}_ then mirror the content to _managed hosts_.
```

Comment:
This section and the following list could use more consistent capitalization.
I would put "external content sources" and "managed hosts" lowercase.